pkg/kubectl/cmd/util/helpers.go

```diff
@@ -62,16 +63,40 @@ func AddSourceToErr(verb string, source string, err error) error {
 	return err
 }
 
+var fatalErrHandler = fatal
```

Comment:
Using an env var will limit the possible functions that I can plug in here. For example if I want to panic() but before I want to collect some metrics or I want to log the error, etc. Maybe we can do both, support some 'pre-defined' functions (panic, fatal, etc..) but also allow to plug in completely custom function.